plasma_framework/migrations/1041_setup_vault.js

```diff
@@ -0,0 +1,179 @@
+        const paymentExitGame = await PaymentExitGame.deployed();
+        const feeExitGame = await FeeExitGame.deployed();
+        const buildDir = path.resolve(__dirname, '../../MultiSigWallet/build/multisig_instance');
+        const gnosisMultisigAddress = fs.readFileSync(buildDir, 'utf8');
+        const gnosisMultisigAbi = { constant: false,
+            inputs: [
+                { name: 'destination', type: 'address' },
+                { name: 'value', type: 'uint256' },
+                { name: 'data', type: 'bytes' },
+            ],
+            name: 'submitTransaction',
+            outputs: [{ name: 'transactionId', type: 'uint256' }],
+            payable: false,
+            type: 'function',
+            signature: '0xc6427474' };
+        // ethVault.setDepositVerifier
+        const setDepositVerifier = web3.eth.abi.encodeFunctionCall(ethVault.abi.find(o => o.name === 'setDepositVerifier'), [ethDepositVerifier.address]);
+        const gnosisSetDepositVerifier = web3.eth.abi.encodeFunctionCall(gnosisMultisigAbi, [ethVault.address, 0, setDepositVerifier]);
+        let transaction = await web3.eth.sendTransaction({ gas: 3000000, to: gnosisMultisigAddress, from: deployerAddress, data: gnosisSetDepositVerifier });
+        console.log(`Submitted transaction with hash for ETH setDepositVerifier: ${transaction.transactionHash}`);
+        let transactionReceipt = null;
+        while (transactionReceipt == null) { // Waiting expectedBlockTime until the transaction is mined
+            transactionReceipt = await web3.eth.getTransactionReceipt(transaction.transactionHash);
+            if (transactionReceipt != null && transactionReceipt.status === true) {
+                console.log('Got a success transaction receipt for ETH setDepositVerifier');
+            } else {
+                await sleep(expectedBlockTime);
+            }
+        }
+        console.log(`Transaction receipt for ETH setDepositVerifier: ${util.inspect(transactionReceipt, { showHidden: false, depth: null })}`);
+        // plasmaFramework.registerVault
+        const registerVault = web3.eth.abi.encodeFunctionCall(plasmaFramework.abi.find(o => o.name === 'registerVault'), [config.registerKeys.vaultId.eth, ethVault.address]);
+        const gnosisRegisterVault = web3.eth.abi.encodeFunctionCall(gnosisMultisigAbi, [plasmaFramework.address, 0, registerVault]);
+        transaction = await web3.eth.sendTransaction({ gas: 3000000, to: gnosisMultisigAddress, from: deployerAddress, data: gnosisRegisterVault });
+        console.log(`Submitted transaction with hash for ETH registerVault: ${transaction.transactionHash}`);
+        transactionReceipt = null;
+        while (transactionReceipt == null) { // Waiting expectedBlockTime until the transaction is mined
+            transactionReceipt = await web3.eth.getTransactionReceipt(transaction.transactionHash);
+            if (transactionReceipt != null && transactionReceipt.status === true) {
```

Comment:
same here `!==` is preferred. Apply to all the  other `if` inside `while` I think